.github/workflows/greetings.yml

```
@@ -1,4 +1,4 @@
-name: Greetings
+ name: Greetings
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)